shared/src/commonMain/kotlin/com/example/travelapp_kmp/listing/ListScreenViewModel.kt

```diff
@@ -27,7 +35,23 @@ class ListScreenViewModel {
             }
         }
     }
+    @OptIn(ExperimentalResourceApi::class)
+    private fun getCountriesSorted(
+        countries: List<Country>,
+        sortOrder: SortOrder
+    ): List<Country> {
+        return if (sortOrder == SortOrder.ASCENDING) {
+            countries.sortedBy { it.name }
+        } else {
+            countries.sortedByDescending { it.name }
+        }.map { country ->
+            country.copy(
+                touristPlaces = country.touristPlaces.sortedBy { it.name }
+            )
+        }
```

Comment:
why map ?

Reply:
There is no need for this. sortBy already give us a list as a result. We just need to return that.

Reply:
This map to sort tourist places by name from A to Z. @SEAbdulbasit